.changeset/calm-masks-rush.md

```diff
@@ -0,0 +1,6 @@
+---
+"wc-vanilla": minor
```

Comment:
Example apps should be changed under 'pie-monorepo' so feel free to remove the 'wc-vanilla' line!